### PR TITLE
Feature/gzip (for #28)

### PR DESCRIPTION
**Pull request by greymd**
* Add feature for compress data.
* Fix the wrong part in test.sh

Let's use this branch for a while.

### CODE REVIEW

File: ttcp.sh [hunk 1]
```diff
@@ -158,6 +158,14 @@ __ttcp::is_env_ok () {
     return 0
 }
 
+__ttcp::encrypt () {
```
Comment (mecab) [inline ttcp.sh]: `encrypt` seems not appropriate name as this part does more than encryption. How about `encode` ?

File: ttcp.sh [hunk 2]
```diff
@@ -158,6 +158,14 @@ __ttcp::is_env_ok () {
...
+    gzip | openssl aes-256-cbc -e -pass pass:$TTCP_PASSWORD
+}
+
+__ttcp::decrypt () {
```
Comment (mecab) [inline ttcp.sh]: how about `decode`

**Review by mecab — CHANGES_REQUESTED** [review]
Good job 👏

Could you consider the function's name as I wrote on inline comments.

**Comment by greymd** [comment]
fixed!

**Review by mecab — APPROVED** [review]

**Comment by mecab** [comment]
LGTM

**Pull request merged**